OpenWord-MMO/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace OpenWord_MMO;
+internal static class OpenGameServer
+{
+    private const byte MaxMessageCharSize = 20;
+    private static readonly IPEndPoint remoteEP = new (IPAddress.Loopback, 3333);
+    private static readonly UdpClient udpClient = new (remoteEP);
+    private static string response = "";
+
+    private static void Main()
+    {
+        Console.WriteLine("Server initializing");
+        Console.WriteLine($"Server set up at port {remoteEP.Port.ToString()}");
+        Console.WriteLine("Server ready & waiting to receive");
+
+        while (true)
+        {
+            IPEndPoint? remoteEndpoint = default;
+            var data = udpClient.Receive(ref remoteEndpoint);
+            var messageString = Encoding.ASCII.GetString(data).Trim();
+            
+            if (data.Length is > MaxMessageCharSize or 0 || messageString.Any(char.IsWhiteSpace))
+            {
+                Console.WriteLine($"Denied message:\n{udpClient}'s message is empty, over 20 characters or contained a whitespace");
+                
+                var errorMessage = Encoding.ASCII.GetBytes("Error. Please adhere to the formatting rules when sending your message");
+                udpClient.Send(errorMessage);
+            }
+            else
+            {
+                Console.WriteLine($"Accepted message");
+                var successMessage = Encoding.ASCII.GetBytes($"Accepted message\nNumber of characters of received message: {data.Length}");
+                udpClient.Send(successMessage);
+            }
+
+            response += " " + messageString;
```

Comment:
technically, this puts a whitespace in the beginning of the "sentence" (when the string is still empty before)